packages/venia-concept/src/components/MiniCart/miniCart.js

```diff
@@ -95,15 +95,15 @@ class MiniCart extends Component {
             <dl className={classes.totals}>
                 <dt className={classes.subtotalLabel}>
                     <span>
-                        Subtotal
-                        {` (${cart.details.items_qty} Items)`}
+                        Cart Total :{` `}
+                        <Price
+                            currencyCode={cartCurrencyCode}
+                            value={cart.totals.subtotal}
+                        />
                     </span>
                 </dt>
                 <dd className={classes.subtotalValue}>
-                    <Price
-                        currencyCode={cartCurrencyCode}
-                        value={cart.totals.subtotal}
-                    />
+                    {` (${cart.details.items_qty} Items)`}
```

Comment:
Could you, please, move the value of this expression to local variable and use this variable here.